test/models/report_test.rb

```diff
@@ -24,7 +24,7 @@ class ReportTest < ActiveSupport::TestCase
     # This includes a row of all zeros, which could only have been generated by populate_category
     assert r.index_data['license'][3][:data].values.all? { |v| v == 0 }
     # Also check that a row with expected data is represented accurately
-    assert_equal r.index_data['license'][1][:data].values, [3, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+    assert_equal r.index_data['license'][1][:data].values, [3, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
One day I'm going to rewrite these tests...

Reply:
One day. I'd do it myself, but every time I look at the `index_data` I turn into Sam Neill's character in Event Horizon.